radicle-cli/src/commands/assign.rs

```diff
@@ -67,7 +69,7 @@ impl Args for Options {
 
         Ok((
             Options {
-                id: id.ok_or_else(|| anyhow!("an issue must be specified"))?,
+                id: id.ok_or_else(|| anyhow!("an issue-id must be specified"))?,
```

Comment:
I'd leave this as it was, it reads better, and we're already specifying the exact type expected in the --help

Reply:
https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-31f756a5a2a4db83a03ed2a78cc3bd1ffbc00693604c06a495f96cc4a73f5f91R72